zebedee/client/client.go

```diff
@@ -119,19 +120,15 @@ func (c *ZebedeeClient) GetDatasetLandingPage(path string) (data.DatasetLandingP
 	return dlp, nil
 }
 
-// SetAccessToken adds an access token to the client to authenticate with zebedee
-func (c *ZebedeeClient) SetAccessToken(token string) {
-	c.accessToken = token
-}
-
-func (c *ZebedeeClient) get(path string) ([]byte, error) {
+func (c *ZebedeeClient) get(ctx context.Context, path string) ([]byte, error) {
 	req, err := http.NewRequest("GET", c.zebedeeURL+path, nil)
 	if err != nil {
 		return nil, err
 	}
 
-	if len(c.accessToken) > 0 {
-		req.Header.Set(common.FlorenceHeaderKey, c.accessToken)
+	if ctx.Value("X-Florence-Token") != nil {
+		accessToken := ctx.Value("X-Florence-Token").(string)
```

Comment:
`go-ns/common/access_token.go` defines this context key already - we should reuse it.

Also this cast could fail and if that happens it would be useful to know its failed. I am not sure if we would want to fail fast on this error but we should log that the value could not be cast as expected.

I think you can do:
```
accessToken, ok := ctx.Value("X-Florence-Token").(string)
if !ok {
  ...
}
```